primo/data_parser/tests/test_well_data_common_methods.py

```diff
@@ -206,7 +211,7 @@ def test_drop_incomplete_data(caplog, get_well_data_from_csv):
         f"Removed a few wells because {col_names.operator_name} information "
         f"is not available for them."
     ) not in caplog.text
-    assert "operator_name" not in wd._removed_rows
+    assert "operator_name" not in wd.get_removed_wells_with_reason
```

Comment:
ARe these name changes primarily to avoid pylint errors? If yes, I would have recommended not spending time fixing them in general. This is OK though, please don't worry about reverting the changes